src/components/colorways/gmk/metropolis.js

```diff
@@ -0,0 +1,42 @@
+export default {
+    name: 'gmk-metropolis',
+    override: {
+        KC_ESC: 'esc',
+        KC_GESC: 'esc',
+        KC_ENT: 'enter',
+        KC_PENT: 'enter',
+        KC_TAB: 'accent1',
+        KC_CAPS: 'accent1',
+        KC_BSPC: 'accent1',
+        KC_PSCR: 'accent1',
+        KC_SLCK: 'accent1',
+        KC_PAUS: 'accent1',
+        KC_INS: 'accent1',
+        KC_HOME: 'accent1',
+        KC_PGUP: 'accent1',
+        KC_DEL: 'accent1',
+        KC_END: 'accent1',
+        KC_PGDN: 'accent1',
+        KC_NLCK: 'accent1',
+        KC_PSLS: 'accent1',
+        KC_PAST: 'accent1',
+        KC_PMNS: 'accent1',
+        KC_PPLS: 'accent1',
+        KC_PCMM: 'accent1',
+        KC_PEQL: 'accent1',
+        KC_LSFT: 'accent2',
+        KC_LCTL: 'accent2',
+        KC_LGUI: 'accent2',
+        KC_LALT: 'accent2',
+        KC_RALT: 'accent2',
+        KC_RGUI: 'accent2',
+        KC_APP: 'accent2',
+        KC_RCTL: 'accent2',
+        KC_RSFT: 'accent2',
+        KC_UP: 'accent2',
+        KC_LEFT: 'accent2',
+        KC_DOWN: 'accent2',
+        KC_RGHT: 'accent2',
+        MO(layer): 'accent3'
```

Comment:
Wrap the key in single quotes and this will work:

```suggestion
        'MO(layer)': 'accent3'
```

![image](https://user-images.githubusercontent.com/18669334/93937781-22cc2e80-fcdd-11ea-8fd9-b8236b431c46.png)

Reply:
done